Program.cs

```diff
@@ -16,7 +16,7 @@ internal class Program
     {
         class Global
         {
-            public static string path = @"C:\Users\sameer.tanveer\source\repos\Project1\Project1\students.json";
+            public static string path = @"C:\Users\sameer.tanveer\Documents\GitHub\Project-1\students.json";
```

Comment:
JSON file should be placed within the project
Also, its path and file name should be stored in the setting file. _What is the setting file name?_